src/microbe_stage/PatchMapGenerator.cs

```diff
@@ -402,8 +402,12 @@ private static void BuildPatchesInRegion(PatchRegion region, Random random)
 
                 // Build seafloor light, using 0m -> 1, 200m -> 0.01, floor to 0.01
                 var sunlightProperty = seafloor.Biome.Compounds[sunlightCompound];
-                sunlightProperty.Ambient = (int)(Mathf.Pow(0.977237220956f, seafloor.Depth[1]) * 100) / 100.0f;
+                var sunlightAmount = (int)(Mathf.Pow(0.977237220956f, seafloor.Depth[1]) * 100) / 100.0f;
+                sunlightProperty.Ambient = sunlightAmount;
+
+                // Need to initialise both to account for loading from saves
                 seafloor.Biome.Compounds[sunlightCompound] = sunlightProperty;
+                seafloor.Biome.CreateSunlight(sunlightAmount);
```

Comment:
I think based on the other code, to make the sunlight more similar to the other biome setup stuff, the sunlight property should be default constructed, and the patch modification code here would then just adjust the light properties (similarly to how the compounds dictionary doesn't need to be created here but is just modified).